.github/ISSUE_TEMPLATE/documentation.md

```diff
@@ -8,10 +8,9 @@ assignees: ''
 
 ## 📚 Documentation
 
-For typos and doc fixes, please go ahead and:
+For typos and fixes please go ahead and: 
 
-1. Create an issue.
-1. Fix the typo.
-1. Submit a PR.
+- for a typo or fix, please send directly a PR (no need to create an issue)
+- if you are not sure about the proper solution, please describe here your finding...
```

Comment:
I meant that there is repetition of `for typo or fix`.

Maybe this whole part should be replaced with something like:
```
For typos or docs fixes, please go ahead and submit a PR (no need to open an issue).
If you are not sure about the proper solution, please describe the issue here...
```